tests/unit/test_archives.py

```diff
@@ -17,6 +17,21 @@ def selectedFiles(self):
         return ['/tmp']
 
 
+@pytest.fixture()
+def archive_env(qapp, qtbot):
```

Comment:
Again, nice change there. Since the fixture name isn't 100% descriptive, could use a docstring?

Reply:
I added a doctoring, and moved this fixture to `unit/conftest.py` since it was used in both `test_diff` and `test_archive`. Additionally, I created a fixture for the integration tests which faced a similar level of code duplication when setting up the archive tab. This fixture was _slightly_ different than the one for unit tests.